src/OpenConext/Profile/Value/Url.php

```diff
@@ -32,9 +32,8 @@
      */
     public function __construct($url)
     {
-        if (filter_var($url, FILTER_VALIDATE_URL) === false) {
-            throw new InvalidArgumentException('URL must be valid: "' . $url . '"');
-        }
+        Assert::string($url, 'URL "%s" must be a string');
```

Comment:
Had to look twice, I think you can drop the first commit of this PR :)

Nice proposal, you've got my approval.